Produlytics/persistence/src/main/java/it/deltax/produlytics/persistence/AccountEntity.java

```diff
@@ -5,44 +5,91 @@
 import javax.persistence.Id;
 import javax.persistence.Table;
 
+/**
+ * Questa classe rappresenta un utente salvato nella banca dati.
+ */
 @Entity
 @Table(name = "account")
 public class AccountEntity {
-	@Id
-	@Column(name = "username", nullable = false)
-	private String username;
+  /**
+   * L'username dell'utente.
+   */
+  @Id
+  @Column(name = "username", nullable = false)
+  private String username;
 
-	@Column(name = "hashed_password", nullable = false)
-	private String hashedPassword;
+  /**
+   * Un hash della password dell'utente.
+   */
+  @Column(name = "hashed_password", nullable = false)
+  private String hashedPassword;
 
-	@Column(name = "administratore", nullable = false)
-	private Boolean administrator;
+  /**
+   * I permessi dell'utente.
+   */
+  @Column(name = "administrator", nullable = false)
+  private Boolean administrator;
 
-	@Column(name = "archived", nullable = false)
-	private Boolean archived;
+  /**
+   * Lo stato di archiviazione dell'utente.
+   */
+  @Column(name = "archived", nullable = false)
+  private Boolean archived;
 
-	protected AccountEntity() {}
+  /**
+   * Costruttore senza argomenti per JPA.
+   */
+  protected AccountEntity() {}
 
-	public AccountEntity(String username, String hashedPassword, Boolean administrator, Boolean archived) {
-		this.username = username;
-		this.hashedPassword = hashedPassword;
-		this.administrator = administrator;
-		this.archived = archived;
-	}
+  /**
+   * Crea una nuova istanza di `AccountEntity`.
+   *
+   * @param username Il valore per il campo `username`.
```

Comment:
Ho scoperto una cosa molto divertente: tu metti i punti alla fine delle frasi e fai cominciare le frasi dei parametri con la maiuscola. Nel back-end abbiamo fatto il contrario...

Reply:
Inoltre io mi sono appena accorto che javadoc non renderizza \`testo\` in monospace 😂

Reply:
Comunque renderizzato sarebbe così https://imgur.com/nmpJJQ1
Non penso ci sia tanta differenza, quindi cambia chi ha meno da cambiare.

Considera però che la prima riga deve finire con un punto o checkstyle segna un warning.

Reply:
> Inoltre io mi sono appena accorto che javadoc non renderizza `testo` in monospace 😂

In che senso? Nell'immagine lo vedo in monospace. Io avevo usato `<code>` ma me lo avete tolto.

Reply:
Comuque proprio perché poi i parametro sono `parameter - <description>` io farei cominciare la descrizione in minuscolo, perché quel `-` è come se fosse un `:` in una frase per me.

Reply:
> In che senso? Nell'immagine lo vedo in monospace.

L'immagine è con {@code testo} invece che con \`testo\`

> Comuque proprio perché poi le descrizioni dei parametri sono parameter - <description> io farei cominciare la descrizione in minuscolo, perché quel - è come se fosse un : in una frase per me.

Ok allora cambio anche io. Devo togliere anche il punto? Mi farebbe un po' strano non non terminare quelle frasi.

Reply:
> Devo togliere anche il punto?

Eh boh. A me in effetti aveva fatto strano quando ho visto che Leila non li aveva messi sui suoi commenti, perché di solito li mettiamo dovunque, ma pensavo che avesse guardato da te. Se guardi la documentazione delle librerie del linguaggio loro mettono i punti nelle descrizioni (la parte sopra per intenderci), ma sulle descrizioni dei parametri no e cominciano con la minuscola lì.

Reply:
> L'immagine è con {https://github.com/code testo} invece che con `testo`

Ah ok. Comunque non so da dove tu abbia preso il testo renderizzato, ma io lo guardavo direttamente appena lo scrivevo, semplicemente facendo l'hover con il mouse. Non avevi controllato la prima volta che hai usato \`\`?

Reply:
> Se guardi la documentazione delle librerie del linguaggio loro mettono i punti nelle descrizioni (la parte sopra per intenderci), ma sulle descrizioni dei parametri no e cominciano con la minuscola lì.

Ok, allora ci adattiamo a quello.

> non so da dove tu abbia preso il testo renderizzato

Su Intellij IDEA vai su Tools > Generate Javadoc

> Non avevi controllato la prima volta che hai usato \`\`?

Eh no 😅 (e in effetti anche lì si nota...)